packages/cli/src/commands/create-plugin/createPlugin.ts

```diff
@@ -405,7 +405,7 @@ const createPlugin = async () => {
     if (ownerIds && ownerIds.length) {
       await addCodeownersEntry(
         codeownersPath!,
-        path.join('plugins', answers.id),
+        path.sep + path.join('plugins', answers.id),
```

Comment:
well frankly i do not know if it even makes sense to use any separator than `/` here but...